src/v2/Components/v2/ArtworkFilter/ArtworkFilters/PriceRangeFilter.tsx

```diff
@@ -71,69 +93,105 @@ export const PriceRangeFilter: React.FC = () => {
   }
 
   const handleSelect = (selectedOption: string) => {
+    if (selectedOption === "custom") {
+      setShowCustom(true)
+      return
+    }
+
+    if (selectedOption !== null) {
+      setCustomRange(parseRange(selectedOption))
+    }
+
+    setShowCustom(false)
     setFilter("priceRange", selectedOption)
-    setCustomRange(parseRange(selectedOption))
   }
 
   return (
-    <Toggle label="Price" expanded>
-      <Flex flexDirection="column" alignItems="left">
-        <RadioGroup
-          deselectable
-          defaultValue={initialRange}
-          onSelect={handleSelect}
-          disabled={atAuction}
-          disabledText="Disabled for biddable works"
-        >
-          {PRICE_RANGES.map((range, index) => (
-            <Radio
-              key={`${index}`}
-              my={0.3}
-              label={<OptionText>{range.name}</OptionText>}
-              value={range.value}
-            />
-          ))}
-        </RadioGroup>
-      </Flex>
-
-      <Clickable
-        mt={1}
-        textDecoration="underline"
-        textAlign="left"
-        onClick={() => setShowCustom(prevShowCustom => !prevShowCustom)}
-      >
-        <Text>{showCustom ? "Hide" : "Show"} custom price</Text>
-      </Clickable>
-
-      {showCustom && (
-        <>
-          <Flex mt={1} alignItems="flex-end">
-            <Input
-              placeholder="$ Min"
-              type="number"
-              min="0"
-              step="1"
-              value={customRange[0]}
-              onChange={handleChange(0)}
-            />
-
-            <Spacer mx={0.5} />
-
-            <Input
-              placeholder="$ Max"
-              type="number"
-              min="0"
-              step="1"
-              value={customRange[1]}
-              onChange={handleChange(1)}
-            />
-          </Flex>
-
-          <Button mt={1} variant="secondaryGray" onClick={handleClick}>
-            Set price
-          </Button>
-        </>
-      )}
-    </Toggle>
+    <>
+      <Toggle label="Price" expanded>
+        {mode === "done" && (
+          <Media lessThan="sm">
+            <Message variant="info" my={2}>
+              Price set; select apply see full results
```

Comment:
typo, should it be "select apply **to** see full results"?

Reply:
Good catch (copied from Figma!)